src/driver/aurora-mysql/AuroraMysqlQueryRunner.ts

```diff
@@ -1918,7 +1918,9 @@ export class AuroraMysqlQueryRunner
                 if (!isAnotherTransactionActive) {
                     await this.rollbackTransaction()
                 }
-            } catch (rollbackError) {}
+            } catch (rollbackError) {
+                /* empty */
```

Comment:
I'm preatty much okay with the previous statement, without additional "/*empty*/" dummy comment.

Reply:
In general we have four options:

1. Remove the empty block complete
2. Dummy `empty` comment (to prevent linter error)
3. Disable only the line with `// eslint-disable-next-line no-empty`
4. Disable the rule 

Which option do you prefer?

Reply:
I'm not sure what "Remove the empty block complete" means, so I would choose option 4.

Reply:
I disabled the rule `no-empty`.